src/main/java/org/prebid/server/health/HealthChecker.java

```diff
@@ -0,0 +1,12 @@
+package org.prebid.server.health;
+
+import java.util.Map;
+
+public interface HealthChecker {
+
+    Map<String, Object> status();
```

Comment:
Unfortunately, we need it as a part of an interface since we call `healthCheckers.forEach(HealthChecker::initialize)` in our StatusHandler Bean.

Or else we need to change how we would initialize our checkers.